### ISSUE TITLE
Wrong episode_title

### ISSUE THREAD

**Issue by OmgImAlexis** [guessit-io/guessit#417]
```
guessit --type movie "The.Hobbit.The.Battle.Of.The.Five.Armies.2014.READ.NFO.NTSC.DVDR-KART3LDVD"
For: The.Hobbit.The.Battle.Of.The.Five.Armies.2014.READ.NFO.NTSC.DVDR-KART3LDVD
GuessIt found: {
    "title": "The Hobbit The Battle Of The Five Armies", 
    "year": 2014, 
    "episode_title": "READ NFO", 
    "other": "NTSC", 
    "format": "DVD", 
    "release_group": "KART3LDVD", 
    "type": "movie"
}
```

**Comment by VeNoMouS** [comment]
technically, that's not wrong, if you look at scene naming standards for labeling tv, that is correct, however when movie type is select there should not be a episode_title processed...

Just my 2 cents..

**Comment by ratoaq2** [comment]
Fixed by #420
